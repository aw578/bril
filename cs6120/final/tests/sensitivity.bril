@main {
  x1: int = const 1;
  x2: int = const 2;
  t3: int = sub x1 x2;
  t4: int = sub x2 x1;
}