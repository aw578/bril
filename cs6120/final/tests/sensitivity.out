Congruence classes for function main:
Class const 1: ['x1.0']
Class const 2: ['x2.0']
Class sub: ['t3.0']
Class sub_new: ['t4.0']
